packages/ra-core/src/dataProvider/useQuery.ts

```diff
@@ -138,12 +151,6 @@ const useQuery = (
     return state;
 };
 
-export interface Query {
-    type: string;
-    resource?: string;
-    payload: object;
-}
-
 export interface QueryOptions {
```

Comment:
This one should be named `UseQueryOptions`. Sorry to find the problems one by one, but the fact that you decided to expose all exports in this file created new problems